charts/fission-all/templates/router/role-kubernetes.yaml

```diff
@@ -1,4 +1,4 @@
-{{- include "kubernetes-role-generator" (merge (dict "namespace" .Values.defaultNamespace "component" "router") .) }}
+{{- include "kubernetes-role-generator" (merge (dict "namespace" .Release.Namespace "component" "router") .) }}
```

Comment:
Probably we should add comment here 

https://github.com/fission/fission/blob/0cfe08df55664575399e7c66b1d0d80e50f331fd/charts/fission-all/templates/_fission-kubernetes-roles.tpl#L357

So in future if we modify permissions we have need to relook into namespace configured.

Reply:
Done